bindata/Makefile

```diff
@@ -0,0 +1,27 @@
+# Mgmt
+# Copyright (C) 2013-2017+ James Shubin and the project contributors
+# Written by James Shubin <james@shubin.ca> and the project contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# The bindata target generates go files from any source defined below. To use
+# the files, import "github.com/purpleidea/mgmt/bindata" and use
+# `bytes, err := bindata.Asset("FILEPATH")` where FILEPATH is the path of the
+# original input file relative to `bindata/`. To make the target you will need
+# to install go-bindata using `go get -u github.com/jteeuwen/go-bindata/...`.
+default:
+	# go-bindata --pkg bindata -o {OUTPUT} {INPUT}
+	go-bindata --pkg bindata -o ./COPYING.go ../COPYING
+	# gofmt the output files
+	gofmt -s -w *.go
```

Comment:
since we'll have one of these targets per output file, have this be specific file. You can use these magic vars throughout :)

https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html